app/jenkins-for-jira-ui/src/components/ConnectionPanel/NotConnectedState.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+import Button from '@atlaskit/button';
+import { ConnectedState } from '../StatusLabel/StatusLabel';
+import {
+	notConnectedStateContainer,
+	notConnectedStateHeader,
+	notConnectedStateParagraph, notConnectedTempImgPlaceholder
+} from './ConnectionPanel.styles';
+
+type NotConnectedStateProps = {
+	connectedState: ConnectedState
+};
+
+const NotConnectedState = ({ connectedState }: NotConnectedStateProps): JSX.Element => {
+	const notConnectedHeader =
+        connectedState === ConnectedState.PENDING ? 'Connection pending' : 'Duplicate server';
+	const notConnectedP1 =
```

Comment:
Are P1 and P2 paragraph 1 and 2? This is a bit confusing. Can we put it all in one <p> tag or rename to make it clearer?